### PR TITLE
Add validation methods to `_Validator` and add it as `XMLSchema` base class

### PR DESCRIPTION
**Pull request by janssenhenning**

### CODE REVIEW

File: lxml-stubs/etree.pyi [hunk 1]
```diff
@@ -531,13 +531,18 @@ class ParseError(LxmlSyntaxError):
     position: Tuple[int, int]
 
 class XMLSyntaxError(ParseError): ...
-class _Validator: ...
+
+class _Validator:
+    def assert_(self, etree: Union[_Element, _ElementTree]) -> None: ...
```
Comment (scoder) [inline lxml-stubs/etree.pyi:536]: `Union[_Element, _ElementTree]` - this seems worth a dedicated typedef by now, e.g. `_ElementOrTree`. There are lots of places in the API that accept either of the two.

Comment (janssenhenning) [reply lxml-stubs/etree.pyi]: Yes, I agree. Done with 31c0159

**Comment by scoder** [comment]
Triggering a fresh CI build after adding Py3.10 etc.

**Comment by scoder** [comment]
Thanks

**Pull request merged**